scripts/build.py

```diff
@@ -1378,9 +1376,20 @@ def generate_build_directory(hashes: Dict[str, str]) -> None:
 
 def generate_python_package() -> None:
     """Generates Python package using setup.py."""
-    print('Building Oppia package...')
-    subprocess.check_call('python setup.py -q sdist -d build', shell=True)
-    print('Oppia package build completed.')
+
+    # We first remove this dev dependencies because they should not be needed
+    # for the package build and we need to verify that they are actually not
+    # needed.
+    try:
+        print('Remove dev dependencies')
+        install_python_dev_dependencies.main(['--uninstall'])
```

Comment:
This will leave the installation tools (pip, pip-tools, setuptools) installed. Is that what we want?

Reply:
Yep these can remain installed.